File: TaskManager/src/EmailObserver.java
public class EmailObserver implements Observer {
    private String email;
    public EmailObserver(String email) {
        this.email = email;
    }
    @Override
    public void update(String message) {
        System.out.println("Sending email to " + email + " with message: " + message);
    }
}

File: taskManager/labfinal_210042111/EmailObserver.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace labfinal_210042111
{
    public class ConsoleObserver:IObserver
    {
        public List<string> notifications = new List<string>();

        public void Update(string message)
        {
            Console.WriteLine("Observer Received: " + message);
            notifications.Add(message);
        }


    }
}
